docs/devs/Use-Jerry-Debugger.md

```diff
@@ -0,0 +1,25 @@
+## Jerry-debugger
+
+Detailed description about the debugger is available
+[here](https://github.com/jerryscript-project/jerryscript/blob/master/docs/07.DEBUGGER.md).
+
+### Enable debugger support in IoT.js
+
+To enable the debugger support under IoT.js, the `--jerry-debugger` option
+should be passed to the `tools/build.py`. The debugger doesn't work with
+jerry-libc therefore this option will implicitly turn on the system libc.
```

Comment:
Yes, you're right, I would just like to emphasize that this option not only enables the debugger interface, but also ensures that the system libc is used by JerryScript. I've modified this part.